### PR TITLE
xmlpull upgrade to have jigsaw and maven metadata fixed

### PR DESCRIPTION
**Pull request by mareknovotny**
Change-Id: I13fe2066d46f564bd6ae827f331e59e5d18016d7

### CODE REVIEW

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build please

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build please

File: pom.xml [hunk 1]
```diff
@@ -333,7 +333,7 @@
     <version.xml-resolver>1.2</version.xml-resolver>
     <version.xmlpull>1.1.3.1</version.xmlpull>
     <version.xmlunit>1.3</version.xmlunit>
-    <version.xpp3>1.1.4c</version.xpp3>
+    <version.xpp3>1.2.0</version.xpp3>
```
Comment (ge0ffrey) [inline pom.xml]: Are you sure the  <version.xmlpull> shouldn't change too?

IIRC, the artifacts xpp3_min and xmlpull need to be upgraded and released together.

Comment (ge0ffrey) [reply pom.xml]: If this isn't a problem, +1 to merge.

Comment (mareknovotny) [reply pom.xml]: good catch, I think I will change it too

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build please

**Comment by mareknovotny** [comment]
can this test failure be related?  @sutaakar @elguardian ?

```
java.lang.AssertionError: expected:<0> but was:<1>
	at org.kie.server.integrationtests.controller.KieControllerCrashIntegrationTest.testCrashAfterRegistered(KieControllerCrashIntegrationTest.java:187)
```

**Comment by sutaakar** [comment]
Looks like a test instability.

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build please

**Comment by mareknovotny** [comment]
all green, @kiegroup/gatekeepers please merge this

**Pull request merged**